### PR TITLE
Print stdout / stderr when container exits with a code.  Resolves #552

### LINKED ISSUE #552: paasta local-run --healthcheck-only does not print output on failure
In this code:

https://github.com/Yelp/paasta/blob/71d2c27a02dd6b1c9784e8e10aef818b45f56600/paasta_tools/cli/cmds/local_run.py#L559-L584

If I'm just running local-run without `--healthcheck` I'll get the output on failure.

However I'll get this (mostly useless) output with --healthcheck-only:

```
$ paasta local-run --service www_pages --healthcheck
...
You're running a container in non-interactive mode.
This is how Mesos runs containers.
Note that some programs behave differently when running with no
tty attached (as your program is about to run).

Running docker command:
docker run '--env="PAASTA_INSTANCE=main"' '--env="PAASTA_SERVICE=www_pages"' '--env="PAASTA_CLUSTER=norcal-devc"' --env=MARATHON_PORT=55972 --env=HOST=dev25-devc.dev.yelpcorp.com --env=MESOS_SANDBOX=/mnt/mesos/sandbox --memory=700m --memory-swap=700m --publish=55972:8888 --name=paasta_local_run_asottile_640229 --volume=/nail/etc/services:/nail/etc/services:ro --volume=/nail/etc/habitat:/nail/etc/habitat:ro --volume=/nail/etc/datacenter:/nail/etc/datacenter:ro --volume=/nail/etc/ecosystem:/nail/etc/ecosystem:ro --volume=/nail/etc/runtimeenv:/nail/etc/runtimeenv:ro --volume=/nail/etc/region:/nail/etc/region:ro --volume=/nail/etc/superregion:/nail/etc/superregion:ro --volume=/nail/etc/topology_env:/nail/etc/topology_env:ro --volume=/nail/srv:/nail/srv:ro --volume=/etc/boto_cfg:/etc/boto_cfg:ro --volume=/var/run/synapse/services/:/var/run/synapse/services/:ro --volume=/nail/etc/zookeeper_discovery/:/nail/etc/zookeeper_discovery/:ro --volume=/nail/etc/kafka_discovery/:/nail/etc/kafka_discovery/:ro --detach=true www_pages-dev-asottile
Found our container running with CID 7604ea0cdf0fbd04e9f4cdd1046afd5fcb27329ad0ccff27a61eece77d3f4a09

Starting health check via http://dev25-devc.dev.yelpcorp.com:55972/status (waiting 60 seconds before considering failures due to grace period):
Healthcheck failed! (disregarded due to grace period)
Container exited with code 22
Detected --healthcheck-only flag, exiting now.

Stopping and removing the old container 7604ea0cdf0fbd04e9f4cdd1046afd5fcb27329ad0ccff27a61eece77d3f4a09...
(Please wait or you may leave an orphaned container.)
...done
```

### PR DESCRIPTION
**Pull request by asottile**

### CODE REVIEW

File: paasta_tools/cli/cmds/local_run.py [hunk 1]
```diff
@@ -577,10 +583,6 @@ def run_docker_container(
             for line in docker_client.attach(container_id, stderr=True, stream=True, logs=True):
                 sys.stdout.write(PaastaColors.grey(line))
         else:
```
Comment (solarkennedy) [inline paasta_tools/cli/cmds/local_run.py]: You still need to print the output if there is no healthcheck_mode and the container isn't running.

Comment (asottile) [inline paasta_tools/cli/cmds/local_run.py]: it is, the `status` check above is unguarded

Comment (asottile) [inline paasta_tools/cli/cmds/local_run.py]: oh I guess no healthcheck_mode... womp

**Comment by asottile** [comment]
should be ready for review again

**Comment by solarkennedy** [comment]
ty, lgtm

**Pull request merged**